akka-stream/src/main/scala/akka/stream/impl/io/InputStreamSinkStage.scala

```diff
@@ -33,7 +33,10 @@ private[akka] object InputStreamSinkStage {
 /**
  * INTERNAL API
  */
-private[akka] class InputStreamSinkStage(readTimeout: FiniteDuration) extends SinkStage[ByteString, InputStream]("InputStreamSink") {
+private[akka] class InputStreamSinkStage(readTimeout: FiniteDuration) extends GraphStageWithMaterializedValue[SinkShape[ByteString], InputStream] {
```

Comment:
Consider it low hanging fruit that I'm friendly enough that I leave for someone else to fix. ;)